packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -544,16 +532,10 @@ Possible alternatives for such technical user authentication are BasicAuthentica
   private async getProviderDestinationService(): Promise<
     DestinationSearchResult | undefined
   > {
-    const provider = await this.getInstanceAndSubaccountDestinations(
+    const destination = await this.fetchDestinationWithoutToken(
       this.providerServiceToken.encoded
     );
-    const destination = this.options.selectionStrategy(
-      {
-        subscriber: emptyDestinationByType,
```

Comment:
[req] The variable `emptyDestinationByType` is still declared in the file and should also be removed.